index.html

```diff
@@ -32,19 +32,17 @@
         </div>
     -->
         <div class="row first teaser-index">
-            <div class="grid-3">
-                <section class="teaser-index-section">
-                    <h3>Installation</h3>
-                    <p>
-                        Check out our <a href="/intro#get-started">guide</a>.
-                    </p>
-                    <br />
-                    <h3>Tutorials</h3>
-                    <p>
-                        Learn in <a href="/intro#get-started">these tutorials</a> how to install Hoodie and get started.
-                    </p>
-                </section>
-            </div>
+          <div class="grid-3">
+              <section class="teaser-index-section">
+                  <h3>Hoodie Camp Release</h3>
+                  <p>
+                      The new Hoodie is around the corner, you can <strong><a href="http://github.com/hoodiehq/hoodie/" target="_blank">give it a try today</a></strong>!
+                  </p>
+                  <p>
+                    <a href="http://github.com/hoodiehq/hoodie/" target="_blank" style="background: transparent;"><img src="/dist1/content_img/index/hoodie-camp-transparent.png" alt="Hoodie Camp" style="border: 0;"/></a>
```

Comment:
yeah I tried to use the CSS but I think our whole setup is borked and I don’t get how it works :( I’m not sure about the alt text, it’s not really adding any valuable content, we might as well set `alt=""` to have it ignored, as it’s just decorative?